src/main/resources/templates/db_scripts/setEffectiveHoldingsLocation.sql

```diff
@@ -1,11 +1,4 @@
-START TRANSACTION;
-
 UPDATE ${myuniversity}_${mymodule}.holdings_record
-  SET jsonb = JSONB_SET(holdings_record.jsonb, '{effectiveLocationId}', TO_JSONB(holdings_record.jsonb->>'temporaryLocationId'))
-WHERE jsonb->>'temporaryLocationId' IS NOT NULL;
-
-UPDATE ${myuniversity}_${mymodule}.holdings_record
-  SET jsonb = JSONB_SET(holdings_record.jsonb, '{effectiveLocationId}', TO_JSONB(holdings_record.jsonb->>'permanentLocationId'))
-WHERE jsonb->>'temporaryLocationId' IS NULL;
-
-END TRANSACTION;
\ No newline at end of file
+  SET jsonb = JSONB_SET(holdings_record.jsonb, '{effectiveLocationId}',
+                        TO_JSONB(COALESCE(temporarylocationid, permanentlocationid)))
```

Comment:
Where do these two values come from? They don't seem to have been retrieved from the JSONB (and don't share the case of these properties).